bee-message/src/output/mod.rs

```diff
@@ -138,6 +138,36 @@ impl Output {
         }
     }
 
+    /// Returns the feature blocks of an `Output`, if any.
+    pub fn feature_blocks(&self) -> Option<&[FeatureBlock]> {
```

Comment:
Can you order these methods please ? NativeTokens, UnlockConditions, FeatureBlocks, ImmutableFeatureBlocks

Reply:
I ordered them alphabetically :cry:

Reply:
Need to be ordered by fields order :)